src/console.rs

```diff
@@ -126,6 +126,14 @@ impl ScreenBufferInfo {
     pub fn max_window_size(&self) -> (i16, i16) {
         (self.0.dwMaximumWindowSize.X, self.0.dwMaximumWindowSize.Y)
     }
+
+    /// Returns the console screen buffer coordinates of the upper-left and
+    /// lower-right corners of the display window.
+    ///
+    /// This corresponds to `srWindow`.
+    pub fn window_rect(&self) -> SMALL_RECT {
```

Comment:
I'd rather not expose winapi types in the public API if we can help it. I _believe_ there are no other such instances in this crate, and I don't think this is worth changing that on its own. Could you define a custom `SmallRect` struct with the corresponding 4 `i16` fields?